src/components/AddEmailForm/AddEmailForm.jsx

```diff
@@ -30,41 +31,53 @@ const _AddEmailForm = ({ emailError, isAdding, wasSaved, add }) => {
   );
 
   const formErrors = emailError ? [emailError] : errors;
+  if (formErrors && formErrors.length) {
```

Comment:
To avoid the &&, trinaries and stuff, I'd do:

```const formErrors = [...(emailError || errors || [])];```

Reply:
To be honest, I would always prefer a ternary over a triple short circuit inside a spread; but I can't argue with the fact that normalizing it wouldn't be better for the multiple places where it's used.

Reply:
At the very least, we could initialize it as an array for default value.